index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Onomeh's Portfolio</title>
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Poppins:regular,bold,500,600&amp;display=swap"
+        />
+      <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      
+   <a href="/" class="logo">My Logo</a>
+              <nav class="navbar">
+                <button class="nav-button" type="button">
+                  <img src="assets/Icons/hamburger-button.svg" alt="more" />
+                </button>
+              </nav>                      
+       
+      </div>
+    
+    <section class="intro">
+      <h1>
+        I'm Onomeh, <br />
+        Glad to see you!
+      </h1>
+      <p class="about-me">
+         I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+      </p>
+      <a href="#" class="connect">LET'S CONNECT</a>
+
+      <ul class="socials-media-icons">
+        <li class="twitter-icon">
+          <a title="twitter" href="https://twitter.com/home"
+            ><img src="./assets/Icons/twitter.svg" alt="twitter-icon"
+          /></a>
+        </li>
+        <li class="linkedin-icon">
+          <a title="linkedin" href="https://www.linkedin.com/in/onwuanibe-onomeh-52300a19a/"><img src="./assets/Icons/linkedin.svg" alt="Linkedin Icon"/></a>
+        </li>
+        <li class="medium-icon">
+          <a title="medium" href="https://medium.com/"
+            ><img src="./assets/Icons/medium.svg" alt="Medium icon"
+          /></a>
+        </li>
+        <li class="github-icon">
+          <a title="github" href="https://github.com/Estherstarr"
+            ><img src="./assets/Icons/github.svg" alt="Github icon"
+          /></a>
+        </li>
+        <li class="angel-icon">
+          <a title="angel" href="#"
+            ><img src="./assets/Icons/angel-list.svg" alt="Angel icon"
+          /></a>
+        </li>
+      </ul>
+    </section>
```

Comment:
- [ ] [OPTIONAL] As it was not marked by the previous review, this is marked as an optional suggestion: 
- Please, consider closing the `body` and `HTML` tags 👍